src/domain/User.js

```diff
@@ -0,0 +1,54 @@
+import { ERROR_MESSAGE } from '../constants/errorMessage.js';
+import { GAME_MESSAGE } from '../constants/gameMessage.js';
+import { MissionUtils } from '@woowacourse/mission-utils';
+
+export default class User {
+
+    async inputForGame() {
+        const carNames = await this.inputCarNames();
+        const count = await this.inputGameCount();
+
+        return { carNames, count };
+    }
+
+    async inputCarNames() {
+        const carNames = await MissionUtils.Console.readLineAsync(GAME_MESSAGE.inputCarNames);
+
+        if (!carNames) {
+            throw new Error(ERROR_MESSAGE.carNameNotUndefinedException);
+        }
+        
+        return this.splitCarNames(carNames);
+    }
+
+    splitCarNames(carNames) {
+        this.checkInputCarNamesError(carNames.split(','));
+
+        return carNames.split(',');
+    }
+
+    checkInputCarNamesError(carNames) {
+        carNames.forEach(name => {
+            if (name === '' || name === ' ') {
+                throw new Error(ERROR_MESSAGE.carNameNotEmptyException);
+            }
+            
+            if (name.length > 5) {
+                throw new Error(ERROR_MESSAGE.carNameLengthOutOf5Exception);
+            }
+        });
+    }
+
+    async inputGameCount() {
+        const count = await MissionUtils.Console.readLineAsync(GAME_MESSAGE.inputGameCount);
+        this.checkInputGameCountError(count);
+
+        return count;
+    }
+
+    checkInputGameCountError(count) {
+        if (Number.isNaN(count)) {
+            throw new Error(ERROR_MESSAGE.numberFormatException);
+        }
```

Comment:
소수와 같은 double형이 입력되었을 때의 예외사항도 있다고 가정한다면 다음처럼 작성하는 것도 괜찮아 보여요!
(물론 요건 따로 예외사항을 빼는게 나을 것 같긴 하네요)
```suggestion
        if (Number.isNaN(parseInt(count))) {
            throw new Error(ERROR_MESSAGE.numberFormatException);
        }
```

Reply:
하나 배워갑니다~!